src/lib/utils/editPath.ts

```diff
@@ -2,4 +2,7 @@ import { join } from 'path'
 
 import { CONTENT_DIR, EDIT_CONTENT_URL } from '@/lib/constants'
 
-export const getEditPath = (relativePath: string): string => join(EDIT_CONTENT_URL, CONTENT_DIR, relativePath, "index.md")
+export const getEditPath = (relativePath: string): string => {
+  const { href } = new URL(join(CONTENT_DIR, relativePath, "index.md"), EDIT_CONTENT_URL)
```

Comment:
```suggestion
return new URL(join(CONTENT_DIR, relativePath, "index.md"), EDIT_CONTENT_URL)
```

Reply:
@nhsz This will give us the entire URL object... wouldn't we want to just return the path here?

<img width="555" alt="image" src="https://github.com/ethereum/ethereum-org-website/assets/54227730/61262232-bbf4-40fb-a981-0c6625d35e1a">

Reply:
missing the `.href` at the end

Reply:
Oh okay... fairly indifferent myself, added for readability but can switch it over